python/samples/getting_started_with_processes/step03/step03b_food_ordering.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+
+import asyncio
+
+from samples.getting_started_with_processes.step03.models.food_order_item import FoodItem
+from samples.getting_started_with_processes.step03.processes.single_food_item_process import SingleFoodItemProcess
+from semantic_kernel.connectors.ai.open_ai.services.open_ai_chat_completion import OpenAIChatCompletion
+from semantic_kernel.kernel import Kernel
+from semantic_kernel.processes.kernel_process.kernel_process_event import KernelProcessEvent
+from semantic_kernel.processes.local_runtime.local_kernel_process import start
+
+################################################################################################
+# Demonstrate the creation of KernelProcess and eliciting different food related events.       #
+# For visual reference of the processes used here check the diagram in:                        #
+# https://github.com/microsoft/semantic-kernel/tree/main/python/samples/                       #
+# getting_started_with_processes#step03b_food_ordering                                         #
+################################################################################################
+
+
+def _create_kernel_with_chat_completion(service_id: str) -> Kernel:
+    kernel = Kernel()
+    kernel.add_service(OpenAIChatCompletion(service_id=service_id), overwrite=True)
+    return kernel
+
+
+async def use_prepare_food_order_process_single_item(food_item: FoodItem):
+    kernel = _create_kernel_with_chat_completion("sample")
+    kernel_process = SingleFoodItemProcess.create_process().build()
+    async with await start(
+        kernel_process,
+        kernel,
+        KernelProcessEvent(id=SingleFoodItemProcess.ProcessEvents.SingleOrderReceived, data=food_item),
+    ) as running_process:
+        return running_process
+
+
+async def use_single_order_fried_fish():
+    await use_prepare_food_order_process_single_item(FoodItem.FRIED_FISH)
+
+
+async def use_single_order_potato_fries():
+    await use_prepare_food_order_process_single_item(FoodItem.POTATO_FRIES)
+
+
+async def use_single_order_fish_sandwich():
+    await use_prepare_food_order_process_single_item(FoodItem.FISH_SANDWICH)
+
+
+async def use_single_order_fish_and_chips():
+    await use_prepare_food_order_process_single_item(FoodItem.FISH_AND_CHIPS)
+
+
+if __name__ == "__main__":
+    asyncio.run(use_single_order_fried_fish())
```

Comment:
[nit] add divider print between orders or since this is not a UT like .net it could also be a script that asks for input with argparse?

something like:
```python
parser = argparse.ArgumentParser(description="Food Ordering sample")  
parser.add_argument(  
      '--order_item',   
      type=FoodItem,   
      help="Choose a food item to prepare"  
)  
args = parser.parse_args()  
print(f"Preparing: {args.order_item.to_friendly_string()}") 
await use_prepare_food_order_process_single_item(args.order_item)
```

Reply:
could be useful to start involving user interaction from early samples so we can build on top of that:
v1: user inputs fish_sandwich
v1++: user chats with assistant and then order_item x get's prepared

Reply:
Good call. Will go with something like this for now:

```
=== Start 'use_single_order_fried_fish' ===
DISPATCH_SINGLE_ORDER: Dispatching 'Fried Fish'!
GATHER_INGREDIENT: Gathered ingredient Fish
CUTTING_STEP: Ingredient Fish has been chopped!
FRYING_STEP: Ingredient Fish is ready!
=== End 'use_single_order_fried_fish' ===

etc...
```